### PR TITLE
Add doc-builder to the repo to add the course on HF

### PR DESCRIPTION
**Pull request by merveenoyan**

### CODE REVIEW

**Comment by MKhalusova** [comment]
For external repos there may be additional changes needed, and it's probably best to ask @mishig25 for a review, not me.

**Comment by merveenoyan** [comment]
@MKhalusova I think there doesn't need to be any changes, I asked Mishig initially and he said the process is the same (I think GH asked for review automatically so)

**Comment by merveenoyan** [comment]
@johko can't ask Mishig for review, is it due to repo settings?

**Comment by merveenoyan** [comment]
@mishig25 can you review this?

File: .github/workflows/build_documentation.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    branches:
+      - main
+
+jobs:
+  build:
+    uses: huggingface/doc-builder/.github/workflows/build_main_documentation.yml@main
+    with:
+      commit_sha: ${{ github.sha }}
+      package: computer-vision-course
+      package_name: computer-vision-course
```
Comment (mishig25) [inline .github/workflows/build_documentation.yml:14]:
```suggestion
      package_name: computer-vision-course
      repo_owner: johko
```

File: .github/workflows/build_pr_documentation.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  cancel-in-progress: true
+
+jobs:
+  build:
+    uses: huggingface/doc-builder/.github/workflows/build_pr_documentation.yml@main
+    with:
+      commit_sha: ${{ github.event.pull_request.head.sha }}
+      pr_number: ${{ github.event.number }}
+      package: computer-vision-course
+      package_name: computer-vision-course
```
Comment (mishig25) [inline .github/workflows/build_pr_documentation.yml:17]:
```suggestion
      package_name: computer-vision-course
      repo_owner: johko
```

**Comment by merveenoyan** [comment]
@johko can you review and merge this?

**Review by johko — APPROVED** [review]
👍

**Pull request merged**